pages/manager_page.py

```diff
@@ -0,0 +1,70 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from pages.locators import AllLocators
+
+
+class ManagerPage:
+    def __init__(self, browser):
+        self.browser = browser
+        self.url = "https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager"
+
+    def open(self):
+        self.browser.get(self.url)
+        element_present = EC.visibility_of_element_located((By.XPATH, "//button[@ng-class='btnClass1']"))
+        WebDriverWait(self.browser, 5).until(element_present)
```

Comment:
Метод `open` должен только открывать страницу по заданному `url`.
```suggestion
    def open(self):
        self.browser.get(self.url)
```